Apps/AdminWebClient/src/Server/Delegates/IImmunizationAdminDelegate.cs

```diff
@@ -0,0 +1,44 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Admin.Server.Delegates
+{
+    using System;
+    using System.Threading.Tasks;
+    using HealthGateway.Common.Models;
+    using HealthGateway.Common.Models.PHSA;
+
+    /// <summary>
+    /// Provides access to Administrative Immunization data.
+    /// </summary>
+    public interface IImmunizationAdminDelegate
```

Comment:
This looks very similar to the ```IImmunizationDelegate```. What do you think about enhancing that interface so that their methods take a ```PatientModel``` or something like that and the implementation uses that however it sees fit (either phn +dob OR hdid)?

Reply:
I modified the local interface but we have a few complexities, the big one right now is that PHSA returns a different model as the result.  We should discuss and create a ticket to clean up and ask PHSA to change.